narwhal/node/src/primary_node.rs

```diff
@@ -49,7 +49,7 @@ impl PrimaryNodeInner {
     /// The window where the schedule change takes place in consensus. It represents number
     /// of committed sub dags.
     /// TODO: move this to node properties
-    const CONSENSUS_SCHEDULE_CHANGE_SUB_DAGS: u64 = 300;
+    const CONSENSUS_SCHEDULE_CHANGE_SUB_DAGS: u64 = 10_000;
```

Comment:
Did it end up providing better results in private testnet?

Reply:
10k is too much, trying again with 1k

Reply:
[not more stable,](https://mysten.grafana.net/d/FaNbUhene/sui-validators-narwhal?orgId=1&var-Environment=mysten-metrics-internal&var-network=private-testnet&var-validator=All&from=1679629233337&to=1679632205909&showCategory=Thresholds&viewPanel=233) reverting to 300